edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_courses.scss

```diff
@@ -1,604 +1,767 @@
-    @include padding(($baseline*2), ($baseline/2), 0, ($baseline/2));
-
-    .courses {
-      @include rtl() { $layout-direction: "RTL"; }
-
-      .courses-listing .courses-listing-item {
-        @include fill-parent();
-
-        margin: ($baseline*0.75) 0 ($baseline*1.5) 0;
-        max-height: $course-card-height;
-      }
-
-      // CASE: Course Discovery Search enabled
-      &:not(.no-course-discovery) {
-        @include span-columns(9);
-
-        @include media($bp-medium) {
-          @include span-columns(8);
-        }
-
-        @include media($bp-large) {
-          @include span-columns(8);
-        }
-
-        @include media($bp-huge) {
-          @include span-columns(9);
-        }
-
-        .courses-listing .courses-listing-item {
-          @include media($bp-medium) {
-            @include span-columns(4); // 4 of 8
-            @include omega(2n);
-          }
-
-          @include media($bp-large) {
-            @include span-columns(6); // 6 of 12
-            @include omega(2n);
-          }
-
-          @include media($bp-huge) {
-            @include span-columns(4); // 4 of 12
-            @include omega(3n);
-          }
-        }
-      }
-
-      // CASE: Course Discovery Search disabled
-      &.no-course-discovery {
-        @include span-columns(12);
-
-        @include media($bp-medium) {
-          @include span-columns(8); // 8 of 8
-          @include omega(2n);
-        }
-
-        @include media($bp-large) {
-          @include span-columns(12);
-        }
-
-        @include media($bp-huge) {
-          @include span-columns(12);
-        }
-
-        .courses-listing .courses-listing-item {
-          @include media($bp-medium) {
-            @include span-columns(4); // 4 of 8
-            @include omega(2n);
-          }
-
-          @include media($bp-large) {
-            @include span-columns(4); // 4 of 12
-            @include omega(3n);
-          }
-
-          @include media($bp-huge) {
-            @include span-columns(3); // 3 of 12
-            @include omega(4n);
-          }
-        }
-      }
-    }
-  }
+.courses-container .courses .course:hover .cover-image::before, .courses-container .courses .course:focus .cover-image::before {
+ opacity:0;
+}
+
+.btn-learnmore {
+  border-style: solid;
+  border-radius: 0.1875rem;
+  border-width: 1px;
+  box-shadow: none;
+  padding: 0.625rem 1.25rem;
+  font-size: 16px;
+  font-weight: normal;
+  text-shadow: none;
+  color: $inverse;
+  background-color: $primary;
+  margin: 10px 15px 20px 15px;
+  text-align: center;
+}
+
+.btn-learnmore:hover {
+  color: $primary;
+  background-color: $inverse;
 }
 
-// +Hero - Home Header
-// ====================
+
+.courses-container .courses .course .course-info h2 {
+  margin-bottom: 5px;
+}
+
+.courses-container .courses .course .course-info .course-date {
+  margin-bottom: 5px;
+}
+
+.courses-container .courses .course .course-info .course-title {
+//  margin: 10px 0 15px 0;
```

Comment:
Remove it if you no longer need it. @luismorenolopera

Reply:
done